lib/win32/windows/time_calc_helper.rb

```diff
@@ -0,0 +1,120 @@
+module Windows
+  module TimeCalcHelper
+
+    # Returns actual no of days for given month;
+    # Array with a 0 is defined to give actual result without
+    # any manipulation. eg, DAY_OF_MONTH[1] = 31
+    # 0(NUMBER) is kept to avoid exceptions during calculations
+    DAY_OF_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
```

Comment:
probably rename this to DAYS_IN_A_MONTH?

Reply:
Done; Changes will be reflected in next commit (along with test cases)
Thank you